react/src/components/molecules/EmergencyAlert/style.scss

```diff
@@ -0,0 +1,43 @@
+@import "global";
+@import '00-base/mixins';
+@import '02-molecules/emergency-alert';
+
+@mixin ma-emergency-alert ($color) {
```

Comment:
can you move lines 5-43 into the assets directory file that you are importing above (line 3) i.e. `@import '02-molecules/emergency-alert';`